src/routes/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Home from '../containers/Home';
+import Global from '../containers/Global';
+import Country from '../containers/Country';
+import TakeCare from '../containers/TakeCare';
+import Error404 from '../containers/Error404';
+import Layout from '../components/Layout';
+
+const App = () => (
+  <Router>
+    <Layout>
+      <Switch>
+        <Route exact path='/' component={Home} />
+        <Route exact path='/global' component={Global} />
```

Comment:
thinking about routes, maybe the global should be the Home too no?  so are the same container on 2 paths, what do you think?

Reply:
Ok, that is a good idea, can we handle the map component visualization with the match of the URL?